spyder/plugins/workingdirectory/plugin.py

```diff
@@ -61,11 +61,14 @@ class WorkingDirectory(SpyderPluginV2):
     def get_name():
         return _('Working directory')
 
-    def get_description(self):
-        return _('Set the current working directory for various plugins.')
+    @staticmethod
+    def get_description():
+        return _("Display and manage the current working directory used by "
+                 "other plugins in Spyder.")
```

Comment:
```suggestion
        return _("Manage the current working directory used in Spyder.")
```

* Elide unnecessary, duplicate and out of place word "Display" (which has a subject of the plugin rather than the user like "Manage" does)
* Eliminate unnecessary and potentially confusing lower-level API details